tests_integration_with_local_chain/markets/omen/test_stealing_on_markets.py

```
@@ -0,0 +1,214 @@
+    balance_after_removing_funding_A, balance_after_removing_funding_B = (
+        get_balances(api_keys_A.bet_from_address, local_web3).total,
+        get_balances(api_keys_B.bet_from_address, local_web3).total,
+    )
+    assert (
+        balance_after_market_creation_A
+        < balance_after_removing_funding_A
+        < starting_balance_A
+    ), "Balance after removing the liquidity should be higher than after market creation (because some liquidity can be withdrawn right away), but lower than before market creation (because some liquidity is now locked for the attacker's bet)"
+
+    # Buying or selling tokens after the liquidity is removed will fail.
+    with pytest.raises(Exception) as e_buying:
```

Ruff

Local variable `e_buying` is assigned to but never used

Remove assignment to unused variable `e_buying`

(F841)